errorsHandlers/errors.go

```diff
@@ -0,0 +1,13 @@
+package errorsHandlers
+
+func CheckNilErr(err error) {
```

Comment:
Every exported stuff (function, struct, variable, etc) should have a description
```go
// CheckNilErr - Description here....
func CheckNilErr(err error) {
```